src/controllers/preference.controller.js

```diff
@@ -49,3 +31,19 @@ exports.details = async function(req, res) {
   const preference = await Preference.findOne({ _id: req.params.id });
   res.status(200).json({ preference });
 };
+
+// exports.create = async function(req, res) {
```

Comment:
just wondering why all the extra comments?

Reply:
I removed a lot of routes, but I want to save them in case we need to re-enable them at some point